main.go

```diff
@@ -23,9 +23,23 @@ func main() {
 		log.Fatal(err)
 	}
 
+	bookUrlPathElements := strings.Split(anyflipURL.Path, "/")
+	// secect only 1st and 2nd element of url to avoid mobile on online.anyflip urls
+	// as path starts with / offset index by 1
+	anyflipURL.Path = path.Join("/", bookUrlPathElements[1], bookUrlPathElements[2])
+
 	downloadFolder := path.Base(anyflipURL.String())
 	outputFile := path.Base(anyflipURL.String()) + ".pdf"
 
+	// parse --extract_title to automatically rename pdf to it's title from anyflip
+	if len(os.Args) > 2 && os.Args[2] == "--extract_title" {
```

Comment:
The flag library is a better alternative than checking flags manually. In addition, it will automatically add a help flag and you can add descriptions to the flags. 
However, I think it would be better to make this the default instead of having to enable it with a flag. I will probably add a flag in the future so that the user can change the name, in case one doesn't want to use the book title from anyflip.